data/LVSMix.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "Luis Ventocilla",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "JUG leader @javabsas. Backend Engineer. Me apasiona las tecnologias backend y el desarrollo en varias plataformas.",
+  "avatar": "https://twitter.com/lvsmix/photo"
```

The following are no longer necessary:
```suggestion
  "bio": "JUG leader @javabsas. Backend Engineer. Me apasiona las tecnologias backend y el desarrollo en varias plataformas.",
```